serialize.js

```diff
@@ -18,7 +18,7 @@ const { JSDOM } = jsdom; // ughhh, why?
 const domNode = require('./dom-node');
 const subtree = require('./subtree');
 
-module.exports = function serialize(domLike) {
+module.exports = function serialize(domLike, iframe = true) {
```

Comment:
I prefer when boolean parameters have names that designate that they're boolean, e.g. `includeIframes` or `inlineIframes` or something like that.